api/projects/serializers.py

```diff
@@ -44,6 +44,15 @@ class Meta:
             "stale_flags_limit_days",
             "edge_v2_migration_status",
         )
+        read_only_fields = (
+            "enable_dynamo_db",
+            "edge_v2_migration_status",
+        )
+
+    def update(self, instance: Project, validated_data: dict) -> Project:
+        # Prevent updates to `organisation` field
+        validated_data.pop("organisation", None)
+        return super().update(instance, validated_data)
```

Comment:
I think it's perhaps better to just add a new serializer for update requests which adds organisation to the read only fields?